### PR TITLE
docs: Explain how to destroy an installation TDE-1276

### PR DESCRIPTION
**Pull request by l0b0**
#### Motivation

Add and clarify documentation.

#### Checklist

- [ ] Tests updated
- [x] Docs updated
- [x] Issue linked in Title

### CODE REVIEW

File: docs/infrastructure/destroy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# How to destroy an installation
+
+Destroying the cluster and stack is not easy, because we use some custom EKS resources to link the two together. Based on a teardown, at time of writing the following sequence should work:
+
+1. Delete the cluster:
+
+   ```bash
+   aws eks wait delete-cluster --name=Workflows
```
Comment (blacha) [inline docs/infrastructure/destroy.md:8]: whats the difference between waiting on `delete-cluster` and then waiting for `cluster-deleted` ?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-        - "cdk8s"
-        - "cdk8s-cli"
-        - "cdk8s-plus-*"
-        - "constructs"
-  ignore:
-    - dependency-name: "@aws-sdk/*"
-      update-types: ["version-update:semver-patch"]
-    - dependency-name: "@types/node"
-      update-types: ["version-update:semver-patch"]
+  - package-ecosystem: 'github-actions'
```
Comment (blacha) [inline .github/dependabot.yml:3]: I wonder if we need to update the CI process to detect/format these files? Im assuming this was something like prettier just autofixing everything?

File: docs/infrastructure/destroy.md (hunk 1 above)
Comment (l0b0) [reply docs/infrastructure/destroy.md:8]: The first command exits immediately, not when the cluster has been deleted. Hence the wait.

File: .github/dependabot.yml (hunk 2 above)
Comment (l0b0) [reply .github/dependabot.yml:3]: Yes, that's shown in the commits.

File: docs/infrastructure/destroy.md (hunk 1 above)
Comment (blacha) [reply docs/infrastructure/destroy.md:8]: was more meaning, `delete-cluster` is a command to delete a cluster vs `cluster-deleted`

```
$ aws eks wait delete-cluster

aws: error: argument subcommand: Invalid choice, valid choices are:
```

File: .github/dependabot.yml (hunk 2 above)
Comment (blacha) [reply .github/dependabot.yml:3]: commits get squashed, it is helpful to mention it in the modification section of a pr

File: docs/infrastructure/destroy.md (hunk 1 above)
Comment (l0b0) [reply docs/infrastructure/destroy.md]: D'oh, fixed!

**Review by blacha — APPROVED** [review]

**Pull request merged**